examples/viewer.py

```diff
@@ -193,6 +193,7 @@ def reconfigure_sim(self) -> None:
         self.active_scene_graph = self.sim.get_active_scene_graph()
         self.default_agent = self.sim.get_agent(self.agent_id)
         self.agent_body_node = self.default_agent.scene_node
+        self.agent_body_node.translation = [2.75, 0.5, 4.0]
```

Comment:
Probably shouldn't hardcode the agent's position in the viewer. Don't know what scene users will load.